hana/exporter.sls

```diff
@@ -9,9 +9,15 @@
 
 hanadb_exporter:
   pkg.installed
```

Comment:
I think colons are missing here

Reply:
the colons are needed if you add `name` later, but here since hanadb_exporter is the argument of pkg.installed we don't need them.. 

https://docs.saltstack.com/en/latest/ref/states/all/salt.states.pkg.html

Reply:
or maybe, i might need to setup vagrant for that..